service-tests/pypi.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+const { isSemver } = require('./helpers/validators');
+const isCommaSeperatedPythonVersions = Joi.string().regex(/^([0-9]+.[0-9]+[,]?[ ]?)+$/);
+const isPsycopg2Version = Joi.string().regex(/^v([0-9][.]?)+$/);
+
+const t = new ServiceTester({ id: 'pypi', title: 'PyPi badges' });
+module.exports = t;
+
+
+/*
+  tests for downloads endpoints
+
+  Note:
+  Download statistics are no longer available from pypi
+  it is exptected that the download badges all show
+  'no longer available'
+*/
+t.create('daily downloads (expected failure)')
+  .get('/dd/djangorestframework.json')
+  .expectJSONTypes({ name: 'downloads', value: 'no longer available' });
```

Comment:
These can all be `.expectJSON()` instead of `.expectJSONTypes()`. Sorry to insist on this. If they don't all conform, invariably it's the nonconforming one the next developer will come along and copy…

Would be great to use something like [Danger](http://danger.systems/js/) to catch this automatically.